oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/endpoint/WebClientReactiveClientCredentialsTokenResponseClient.java

```diff
@@ -58,6 +59,8 @@
 			String tokenUri = clientRegistration.getProviderDetails().getTokenUri();
 			BodyInserters.FormInserter<String> body = body(authorizationGrantRequest);
 
+			org.springframework.web.reactive.function.client.WebClient.RequestBodyUriSpec spec = webClient.post();
```

Comment:
Can we change this to use an import rather than in-lining the full package name?